src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -743,6 +744,15 @@ class OrganizationEventsTraceEndpointBase(OrganizationEventsV2EndpointBase):
         "GET": ApiPublishStatus.PRIVATE,
     }
 
+    def get_projects(self, request: Request, organization, project_ids=None, project_slugs=None):
+        return super().get_projects(
+            request,
+            organization,
+            project_ids={-1},
+            project_slugs=None,
+            include_all_accessible=True,
+        )
```

Comment:
A comment explaining why this override is necessary will be helpful here as it's not immediately obvious why this is needed.